pkg/kubelet/kubelet.go

```diff
@@ -298,6 +304,9 @@ func NewMainKubelet(
 		resolverConfig:                 resolverConfig,
 		cpuCFSQuota:                    cpuCFSQuota,
 		daemonEndpoints:                daemonEndpoints,
+		authenticator:                  authenticator,
+		authorizerAttributeGetter:      authorizerAttributeGetter,
```

Comment:
Given current structure, this will require a single `AttributeGetter` that can interpret all kubelet request URLs, correct?